slack_bolt/request/async_internals.py

```diff
@@ -41,6 +44,15 @@ def build_async_context(
     channel_id = extract_channel_id(body)
     if channel_id:
         context["channel_id"] = channel_id
+    function_execution_id = extract_function_execution_id(body)
+    if function_execution_id:
+        context["function_execution_id"] = function_execution_id
+        function_bot_access_token = extract_function_bot_access_token(body)
+        if function_bot_access_token is not None:
+            context["function_bot_access_token"] = function_bot_access_token
+        function_inputs = extract_function_inputs(body)
+        if function_inputs is not None:
```

Comment:
I wonder how this code would react to a custom step invocation that accepts zero inputs 🤔

Reply:
From my testing a custom step execution payload with no input parameter will have the value `{}` (empty dictionary)
`function_inputs` will never be `None` when handling custom step related events 🟢